### ISSUE TITLE
runtests output verbosity 

### ISSUE THREAD

**Issue by sampeka** [inonit/drf-haystack#40]
Hey there, first of all thanks for the work on this. Theres definitely a need for this library and I'm looking forward to working with it :) 

I'd like to contribute toward the library, but I'm finding the output from the tests quite confusing and hard to follow. Even when removing the `--verbose` flag the output is incredibly verbose, seemingly dumping the contents of the elasticsearch index. Is there a way to stop this behaviour?

**Comment by sampeka** [comment]
Resolves this by just editing the settings.py file, not sure if the `--verbose` flag does anything?

**Comment by rhblind** [comment]
Hi,
Thanks for the kind words =)
Yeah, I agree the output is quite noisy when tests fail, but I haven't really looked into it. Did you find a fix?

**Comment by sampeka** [comment]
I raised the log level for the `elasticsearch` and `elasticsearch.trace` loggers in tests.settings, might be worth keeping them above debug by default.

**Comment by rhblind** [comment]
Yeah, thanks for the tip =)
